src/server-settings.ts

```diff
@@ -35,7 +36,11 @@ type SettingsRules = {
 }
 
 const settingsRules: SettingsRules = {
-
+  'logo_url' : {
+    description: 'The application logo to display on the a12nserver pages',
+    fromDb: true,
+    default: '' //curveball logo
```

Comment:
The curveball logo is already part of this codebase. After logging in, you should see it in the top-left. So this value should be something like: `/_hal-browser/assets/logo.svg` (I don't know the actual path).